production/mediapublic

```diff
@@ -0,0 +1,110 @@
+
+##
+# You should look at the following URL's in order to grasp a solid understanding
+# of Nginx configuration files in order to fully unleash the power of Nginx.
+# http://wiki.nginx.org/Pitfalls
+# http://wiki.nginx.org/QuickStart
+# http://wiki.nginx.org/Configuration
+#
+# Generally, you will want to move this file somewhere, and start with a clean
+# file but keep this around for reference. Or just disable in sites-enabled.
+#
+# Please see /usr/share/doc/nginx-doc/examples/  for more detailed examples.
+##
+
+upstream api {
+  server 127.0.0.1:6543;
+}
+
+server {
+  listen 80 default_server;
+  listen 8080 default_server;
+  listen [::]:80 default_server ipv6only=on;
+
+  error_log  /var/log/nginx/error.log  warn;
+
+  server_name _;
+
+  root /var/www/mediapublic/client/public;
+  index index.html;
+
+  location /api {
+        rewrite /api(.*) $1  break;
+        proxy_set_header        Host $http_host;
+        proxy_set_header        X-Real-IP $remote_addr;
+        proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header        X-Forwarded-Proto $scheme;
+
+        client_max_body_size    10m;
+        client_body_buffer_size 128k;
+        proxy_connect_timeout   1s;
+        proxy_send_timeout      10s;
+        proxy_read_timeout      10s;
+        proxy_buffering         off;
+        proxy_temp_file_write_size 64k;
+        proxy_pass http://api;
+        proxy_redirect          off;
+  }
+
+  # Expire rules for static content
+
+	# Feed
+	location ~* \.(?:rss|atom)$ {
+	  expires 1h;
+	  add_header Cache-Control "public";
+	}
+
+	# Media: images, icons, video, audio, HTC, fonts
+	location ~* \.(?:jpg|jpeg|gif|png|ico|cur|gz|svg|svgz|mp4|ogg|ogv|webm|htc|otf|woff|woff2)$ {
+	  expires 1M;
+	  access_log off;
+	  add_header Cache-Control "public";
+	}
+
+	# CSS and Javascript
+	location ~* \.(?:css|js)$ {
+	  expires 1y;
+	  access_log off;
+	  add_header Cache-Control "public";
+	}
+
+  location / {
+    # First attempt to serve request as file, then
+    # as directory, then fall back to displaying a 404.
+    try_files $uri $uri/ =404;
+  }
+
+  #error_page 404 /404.html;
+
+  # redirect server error pages to the static page /50x.html
+  #
+  #error_page 500 502 503 504 /50x.html;
+  #location = /50x.html {
+  # root /usr/share/nginx/html;
+  #}
+}
+
+
+# HTTPS server
+#
+#server {
+# listen 443;
+# server_name localhost;
+#
+# root html;
```

Comment:
I'm pro leaving commented code in the code base when in rapid development mode like this.  Later down the line I do agree we should pull it out.